test_suite/tests/derive.rs

```diff
@@ -11,16 +11,18 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#![cfg_attr(not(feature = "std"), no_std)]
 
-use scale_info::prelude::{
-    boxed::Box,
-    marker::PhantomData,
-};
+#![cfg_attr(not(feature = "std"), no_std)]
 
 use pretty_assertions::assert_eq;
+use scale::Encode;
 use scale_info::{
     build::*,
+    prelude::{
+        boxed::Box,
+        marker::PhantomData,
+        Compact,
```

Comment:
I'm not sure we do want `codec::Compact` in our prelude. I think we already have a dependency on `parity_scale_codec` in this test crate which we can use

Reply:
See comment above, happy to remove it. The point was more about signalling than a technical need.